sdk/core/azure-core/test/ut/curl_options_test.cpp

```diff
@@ -365,4 +365,41 @@ namespace Azure { namespace Core { namespace Test {
         0);
   }
 
+  TEST(CurlTransportOptions, disableSslCaching)
+  {
+    Azure::Core::Http::CurlTransportOptions curlOptions;
+    curlOptions.EnableCurlSslCaching = true;
```

Comment:
Hmm, the test is called "disableSslCaching", yet we do `curlOptions.EnableCurlSslCaching = true`, is this intended?
BTW, should we have tests for both `curlOptions.EnableCurlSslCaching = true` and `curlOptions.EnableCurlSslCaching = false`?

Reply:
Yes I will update the test. Since EnableCurlSslCaching is default, all other tests validate the standard path. I can add one more test, but it will be a line for line duplicate of some other basic validation test. Let me know